src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -728,6 +728,33 @@ def get_enable_advanced_network_observability(self) -> Optional[bool]:
             return not disable_advanced_network_observability
         return None
 
+    def get_advanced_networking_observability_tls_management(self) -> Union[str, None]:
+        """Obtain the value of advanced_networking_observability_tls_management.
+
+        :return str or None
+        """
+        tls_management = self.raw_param.get("advanced_networking_observability_tls_management")
+        enable_advanced_network_observability = self.raw_param.get("enable_advanced_network_observability")
+        if tls_management is not None:
+            if (
+                self.mc and
+                self.mc.network_profile is not None and
+                self.mc.network_profile.advanced_networking is not None and
+                self.mc.network_profile.advanced_networking.observability is not None and
+                self.mc.network_profile.advanced_networking.observability.enabled
+            ):
+                return tls_management
+            if (
+                enable_advanced_network_observability is not None and
+                enable_advanced_network_observability
+            ):
```

Comment:
not blocking, but I think this would be enough, enable_advanced_network_observability could a bool or `None` and only `True` would meet the condition
```suggestion
            if enable_advanced_network_observability:
```

Reply:
updated and running [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=100253020&view=results) for test_aks_create_advanced_networking_observability_tls_management and
test_aks_update_advanced_network_observability